bugs/31.zig

```diff
@@ -23,11 +23,15 @@ fn protection(comptime func: anytype, args: anytype) void {
 }
 
 pub fn logInfo(comptime format: []const u8, args: anytype) void {
-    std.io.getStdOut().writer().print(format, args) catch return;
+    var buf: [4096]u8 = undefined;
+    var writer = std.fs.File.stdout().writer(&buf);
```

Comment:
i've heard it recommended that we use instead of 

```
var writer = std.fs.File.stdout().writerStreaming(&buf);
```

because it's not seekable